CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 2.0.0 (10/13/2023)
+- fix: improved fixed effect features [#69](https://github.com/washingtonpost/elex-live-model/pull/69)
+- fix: with the CLI, model-specific parameters passed in as a dictionary [#70](https://github.com/washingtonpost/elex-live-model/pull/70)
+- fix: additional logic in the CLI to find the `.env` file [#71](https://github.com/washingtonpost/elex-live-model/pull/71)
+- feat: all models are now subclasses of `BaseElectionModel` [#72](https://github.com/washingtonpost/elex-live-model/pull/72)
+- feat: ability to create custom estimands [#75](https://github.com/washingtonpost/elex-live-model/pull/75)
+- feat: conformal bootstrap model [#76](https://github.com/washingtonpost/elex-live-model/pull/76)
```

Comment:
let's call it just bootstrap model here, the branch name was unfortunately not right on my end

Reply:
No worries!  Done! 🎉